tests/drivers/crc/testcase.yaml

```
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: Apache-2.0
+
+tests:
+  crc.test:
+    integration_platforms:
+      - nucleo_g474re
+    platform_allow: nucleo_g474re modbuckboost_v2
```

`modbuckboost_v2` is an out-of-tree target so shouldn't be included here

```suggestion
    platform_allow: nucleo_g474re
```